### ISSUE TITLE
Lawyers for Good Government is looking for a volunteer html/js dev to help with their Russia/Trump timeline!

### ISSUE THREAD

**Issue by rapicastillo** [ProgressiveCoders/projects#113]
**From Sam Frank: **

### About the group		
Lawyers for Good Government is looking for a volunteer html/js dev to help with their Russia/Trump timeline! The app is mostly done, we just need help putting in events on the timeline. Over the past 2 months, lawyers from around the country have been compiling evidence of Trump/Russia contact. They're taking that evidence and putting it into an interactive timeline for people to use. 

### Who we're looking for		
The last step is getting those events into the timeline! We could also use some styling of the timeline.

### Relevant Links		

 
(if this is a volunteer opportunity, this will be added to the [volunteer toolkit](http://toolkit.progcode.org/volunteer.php) )
Inspiration: "ISSUE_TEMPLATE", Chi Hack Night (http://www.chihacknight.org), 		
https://github.com/chihacknight/breakout-groups/blob/master/ISSUE_TEMPLATE

**Comment by rapicastillo** [comment]
Awesome @davidfasel ! Will let @sgeofrank know 🙌

**Comment by 243** [comment]
Does this still need a dev?

**Comment by ajhyndman** [comment]
I'd be interested in contributing, too, if that's helpful!

**Comment by sgeofrank** [comment]
Yes! are you in the progcode slack? I'm sgeofrank

**Comment by ajhyndman** [comment]
@sgeofrank Yep, I just messaged in what I think is the relevant slack channel!  I used the same name.
